csuntechlab/iSTART
# STR-255 -- FE: Fix event handlers for checking user inactivity.

# Description
Removed event handler checking for user inactivity; refactored functionality into a mixin and called when updated.
  
# Testing
1. Set the IDLE_TIMEOUT env value to something reasonable (*value represented in minutes*) for testing.
2. Check to see that after login, the user gets booted out if they haven't done anything.
3. Ensure when going through the app like normal, the idle timer functionality resets.
